apps/web/src/features/sequences/components/SequenceCreateModal.tsx

```
@@ -0,0 +1,338 @@
+															>
+																{delay.label}
+															</SelectItem>
+														))}
+													</SelectContent>
+												</Select>
+											</TabsContent>
+
+											<TabsContent value="custom" className="space-y-3">
+												<div className="grid grid-cols-3 gap-3">
+													<div>
+														<label className="font-medium text-sm">Days</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)